privacyidea/static/components/config/views/config.policies.details.html

```diff
@@ -221,6 +221,16 @@ <h3 translate>Condition</h3>
                            placeholder="userA, userB"/>
                 </div>
             </div>
+            <div class="form-group">
+                <label for="user_case_insensitive" class="col-sm-6">
+                    <input type="checkbox" name="user_case_insensitive"
+                           ng-model="params.user_case_insensitive"
+                           id="case-sensitive">
+                    <translate>
+                The username should be case-insensitive.
```

Comment:
In the UI I would probably choose something short like as label like:

"Username case-insensitive"

Reply:
[b7bc726](https://github.com/privacyidea/privacyidea/pull/3852/commits/b7bc726f2279a85bad77918536eb77dbabbf0523)